terraform/deployments/apps/content-store/main.tf

```diff
@@ -30,6 +30,7 @@ module "task_definition" {
   image_tag                        = var.image_tag
   mesh_name                        = var.mesh_name
   mongodb_url                      = "mongodb://${var.mongodb_host}/content_store_production"
+  plek_hostname_prefix             = ""
```

Comment:
is this intended to be empty?

Reply:
Yep!

Reply:
Might be slightly nicer to have it default to empty in variables.tf? Would avoid the surprise that way. This looks slightly like it's overriding something to empty that might not otherwise have been empty.

Reply:
Thanks, I've done that 👍